Changelog

```diff
@@ -1,4 +1,81 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add `Auto resize columns` functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in `Manual` mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when default "temp" path changed (glassez)
+    - FEATURE: Add tuning options related to performance warnings (Chocobo1)
+    - FEATURE: Add right click menu for status filters (An0n)
+    - FEATURE: Allow setting the number of maximum active checking torrents (An0n)
+    - FEATURE: Add option to toggle filters sidebar (AbeniMatteo)
+    - FEATURE: Allow to set `working set limit` on non-Windows OS (Chocobo1)
+    - FEATURE: Add `Export .torrent` action (Chocobo1)
+    - FEATURE: Add keyboard navigation keys (itlezy)
+    - FEATURE: Allow to use POSIX-compliant disk IO type (Coda)
+    - FEATURE: Add `Filter files` field in new torrent dialog (thalieht)
+    - FEATURE: Implement new icon/color theme (now-im)
```

Comment:
@now-im I recall that you made a comment about this. That I should also attribute another person. Was it made in error and you retracted it?

Reply:
@sledgehammer999 Well in #15769 commit, it's written "Co-authored-by: @xavier2k6" https://github.com/qbittorrent/qBittorrent/pull/15769/commits/199d770e152275d01da6134b802f2e16fa4e96af

Most comments hidden away in that merged pull request, but even in one old comment from year 2021 it was already written by @now-im -> "Without you, I would have just made cosmetic changes and core problems would have remained unsolved. You should be equally credited for this PR, if merged.": https://github.com/qbittorrent/qBittorrent/pull/15769#issuecomment-981414364